### ISSUE TITLE
{Suggestion} Turkish Generic Adlist

### ISSUE THREAD

**Issue by ghost** [gorhill/uBlock#637]
Helllo,

I am a Turkish user of uBlock Origin on every browser.
Can you please add Turkish List of Fanboy on the uBlock's 3rd party filters list.
Please see here : https://www.fanboy.co.nz/regional.html
You can see it on the to right. It is secure https connection for filters not http.
I know it is not updated recently but It has good generic filters for Turkish websites.

Thank you again,
keep up the good work.

**Comment by gorhill** [comment]
For a filter list to appear in the set of stock filter lists, my rule is that a list must have been updated in the last 365 days. If it's not the case, it does not qualify. There are actually current stock filter lists that I may have to remove because they are no longer updated.

However I have added Fanboy's Turkish to the [list here](https://github.com/gorhill/uBlock/wiki/Filter-lists-from-around-the-web), so one can subscribe to it easily.
